include/tc/c2/tc_op.h

```diff
@@ -37,11 +37,12 @@ class TcOp : public Operator<Context> {
  public:
   TcOp(const OperatorDef& operator_def, Workspace* ws)
       : caffe2::Operator<Context>(operator_def, ws),
+        tc_(OperatorBase::GetSingleArgument<std::string>("lang", "ERROR")),
+        tcName_(
+            OperatorBase::GetSingleArgument<std::string>("tcName", "ERROR")),
```

Comment:
for clarity, this change is needed because `name` is an arg that python bindings in caffe2 core code uses already. so it conflicts with the name here if I also use tc -caffe2 from python

Reply:
`tcDef` ?

Reply:
it's really a name so we should make it explicit that it's the *name* that user needs to set the value to. When I read `tcDef` it feels to me that I need to pass the tc language which is misleading

Reply:
sorry, I entered the comment in the wrong field.  I meant `tcDef` instead of `def`, not here

Reply:
got it, I'll make the change